src/main/java/io/github/techstreet/dfscript/script/action/ScriptActionType.java

```diff
@@ -119,6 +120,31 @@ public enum ScriptActionType {
             ctx.value("Value")
         ))),
 
+    GET_REQUEST(builder -> builder.name("Get Webrequest")
+        .description("Makes a get request to the internet.")
```

Comment:
With the nature of web requests, would describing how it works (sends only GET, how it fails on response, how it waits) be useful to have in the description?